contrib/aws/aws-sdk-go-v2/aws/aws.go

```diff
@@ -136,10 +139,19 @@ func (mw *traceMiddleware) deserializeTraceMiddleware(stack *middleware.Stack) e
 	}), middleware.Before)
 }
 
-func serviceName(cfg *config, serviceID string) string {
+func getSpanName(awsService, awsOperation string) string {
```

Comment:
https://go.dev/doc/effective_go#Getters
Just name this `func spanName(service, operation string) string`